test/functional/cursor.test.js

```diff
@@ -2839,6 +2840,28 @@ describe('Cursor', function() {
     }
   });
 
+  /**
+   * @ignore
+   */
+  it('shouldSetReadConcern', {
```

Comment:
nit: this compactedCamelCaseForm is from the legacy `integra` runner. Let's call this something more readable for future maintainers. 
```suggestion
  it('should allow setting the readConcern through a builder method', {
```
or something like that

Reply:
Cool, good to know.